_posts/_2021-05-14-opensearch-beta-announcement.markdown

```diff
@@ -81,4 +69,5 @@ If you're excited to jump in, we've marked a few issues that would be really hel
 
 **Questions? Feedback?**
 
-Let us know in the [forums](https://discuss.opendistrocommunity.dev/). We are also having biweekly [Community Meetings](https://www.meetup.com/Open-Distro-for-Elasticsearch-Meetup-Group/). The community meetings will include progress updates at every session as well as leaving time for Q&A. The next community meeting is on May 6, 2021 at 7:30 AM Pacific Time (UTC -8).
+Let us know in the [forums](https://discuss.opendistrocommunity.dev/). We are also having biweekly [Community Meetings](https://www.meetup.com/Open-Distro-for-Elasticsearch-Meetup-Group/). The community meetings will include progress updates at every session as well as leaving time for Q&A. The next community meeting is on May 18, 2021 at
```

Comment:
Let's also link customers to github issues on the opersearch repo as an option to give feedback

Reply:
Great idea - I'll add.

Reply:
Done!